resource/test/ignite_rules.json

```diff
@@ -1474,57 +1972,91 @@
           "WebSite": false,
           "Lab": false
         },
-        "HOL_ILL": {
+        "IDL_Evaluations": {
           "Email": false,
           "Mobile": false,
           "WebSite": false,
-          "Lab": false
+          "Lab": true
         },
-        "TechLeadersForum": {
-          "Email": false,
+        "TEFPreSurvey": {
+          "Email": true,
           "Mobile": false,
           "WebSite": false,
           "Lab": false
         },
-        "PlusPass": {
+        "TEFPostEventSurvey": {
           "Email": false,
+          "Mobile": true,
+          "WebSite": true,
+          "Lab": false
+        },
+        "TEFDeviceSurvey": {
+          "Email": false,
+          "Mobile": true,
+          "WebSite": true,
+          "Lab": false
+        },
+        "PlusPassSurvey": {
+          "Email": true,
           "Mobile": false,
           "WebSite": false,
           "Lab": false
         }
       },
       "SessionSetAccess": [
         {
-          "ApplicableFrom": null,
-          "ApplicableTo": null,
-          "Identifier" : "Lab"
+          "ApplicableFrom": "2016-09-25T00:00:00Z",
```

Comment:
These need to state the timezone or be set to the correct UTC time e.g. 2016-09-25T00:00:00-05:00 or 2016-09-25T05:00:00Z